configure.ac

```diff
@@ -25,12 +25,11 @@ AM_PROG_CC_C_O
 AC_ISC_POSIX
 AC_USE_SYSTEM_EXTENSIONS
 dnl http://www.gnu.org/s/libc/manual/html_node/Feature-Test-Macros.html
-dnl _LARGEFILE_SOURCE: enable support for new LFS funcs (ftello/etc...)
 dnl _LARGEFILE64_SOURCE: enable support for 64-bit variants (off64_t/fseeko64/etc...)
 dnl NB: We do not want -D_FILE_OFFSET_BITS=64 because we need to interpose both 32-bit
 dnl and 64-bit FS interfaces, and having the C library rewrite them makes that difficult.
 dnl Along those lines, we do not use AC_SYS_LARGEFILE.
-AS_VAR_APPEND([CPPFLAGS], [" -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE"])
+AS_VAR_APPEND([CPPFLAGS], [" -D_LARGEFILE64_SOURCE"])
```

Comment:
Why do we still need this?

Reply:
We need off64_t and the xxx64 declarations for use in the wrapper functions. The xxx64 functions are also called explicitly by tests.

We could probably move it to libsandbox/local.mk and tests/local.mk instead of declaring it globally, however.

Reply:
Technically, I guess we don't need to define _LARGEFILE64_SOURCE at all since glibc treats _GNU_SOURCE the same way.

In case glibc ever changes that, I moved the _LARGEFILE64_SOURCE define to the source files where it is actually needed:

libsandbox/wrappers.c
tests/creat64-0.c
tests/fopen64-0.c
tests/mkostemp64-0.c
tests/mkostemps64-0.c
tests/mkstemp64-0.c
tests/mkstemps64-0.c
tests/open64-0.c
tests/openat64-0.c
tests/truncate64-0.c

Reply:
Thanks. I think this is worthwhile given it's both explicit and also means we're sort of guarded against automagic inclusion of it for musl at least (which still, for now, honours `_LARGEFILE64_SOURCE`).